### PR TITLE
#303 enhancement for breeds query

### LINKED ISSUE #303: Enhancement for breeds query
**Goal**
Enhancement for breeds query

**Description**
- [ ] Make species parameter optional. If not specified, return all species breeds
- [ ] Add `speciesId` and `speciesValue` fields

**Expected behaviour**
Such GraphQL query

```gql
{
  breeds(language:"lt") {
    id
    value
    abbreviation
    speciesId
    speciesValue
  }
}
```

Should return all breeds entries for the species, e.g. (only sample):
```gql
{
  "data": {
      "breeds": [
          {
            "id": 12,
            "value": "Afenpinčeriai",
            "abbreviation": "AP",
            "speciesId": 1,
            "speciesValue": Šuo
          }
    ]
}
```

### PR DESCRIPTION
**Pull request by pergaliuke**
Make species parameter optional. If not specified, return all species breeds
Add speciesId and speciesValue fields
Closes #303

### CODE REVIEW

**Comment by pergaliuke** [comment]
I am not sure if I needed to edit Type Breed or create another one. Type Breed is used in animal query as well: 
```
{
  animal(id: 1) {
    details {
      breed {
        abbreviation
        value
        speciesId
      }
      species {
        id
      }
    }
  }
}
```
The result of animal query will be: 
```
{
  "data": {
    "animal": {
      "details": {
        "breed": {
          "abbreviation": "LKT",
          "value": "Leiklendo terjerai",
          "speciesId": null
        },
        "species": {
          "id": 1
        }
      }
    }
  }
}
```

**Comment by AzisK** [comment]
This ticket is about the `breeds` query only, no need to edit the `breed` query

File: src/sql-queries/breed.ts [hunk 1]
```diff
@@ -29,15 +33,15 @@ export const getBreedQuery = (
     const text = `
         SELECT b.id, b.abbreviation, bt.translation as value
         FROM breed b
-        LEFT JOIN breed_translation bt ON bt.breed = b.id
+                 LEFT JOIN breed_translation bt ON bt.breed = b.id
```
Comment (AzisK) [inline src/sql-queries/breed.ts:36]: Why such alignment?

**Review by AzisK — APPROVED** [review]
Great job!

**Comment by pergaliuke** [comment]
Introduced type BreedWithSpecies in order to avoid affect on animal query.

**Pull request merged**